arithmetization/src/main/java/net/consensys/linea/zktracer/module/mul/MulOperation.java

```diff
@@ -473,7 +473,7 @@ private void traceSubOp(Trace trace, int stamp) {
           .exponentBitAccumulator(expAcc)
           .exponentBitSource(isExponentInSource())
           .squareAndMultiply(squareAndMultiply)
-          .bitNum(UnsignedByte.of(getBitNum()))
+          .bitNum(getBitNum())
```

Comment:
`getBitNum` returns an int and `bitNum` requires a unsigned byte.

```java
  public int getBitNum() {
    return bitNum(index, exponentBits.length());
  }
```

where
```java
      case EXPONENT_NON_ZERO_RESULT -> {
        this.exponentBits = new BigInteger(1, arg2.toArray()).toString(2);
        squareAndMultiply = false;
      }
```

Could we make sure that the `toString(2)` will trim the leading zero's if there are any ? If this is not the case, could we add a unit test for it where we provide `arg2` of the form
- `0x 00`
- `0x 00 01`
- `0x 00 01 00 ... 00` with a total of 17 bytes
- `0x 00 01 00 ... 00` with a total of 18 bytes